cobalt/build/configs/linux_common.gn

```diff
@@ -113,3 +113,6 @@ icu_use_data_file = false
 
 # Disable the Pepper API (PPAPI) plugin support
 enable_ppapi = false
+
+# Enable the compiler size optimizations e.g. -Os.
+optimize_for_size = true
```

Comment:
I couldn't find any more instances of `optimize_for_size` in this
folder (`main` and `26.eap` branches) (I'm guessing AndroidTV adds
it in the internal build?) Could/should we add this gn variable to
`common.gni` instead for all platforms to benefit from it?